### PR TITLE
Dz/docs

### PR DESCRIPTION
**Pull request by DZuz14**
I am not too familiar with async await, so I am not exactly sure how to implement that, so that is not included.

### CODE REVIEW

**Comment by DZuz14** [comment]
@jasonmerino Yay or nay for the examples? I'd be glad to change them up, and put whatever you want in there. Figured some of the examples will help newcomers to React Native as a whole.

File: README.md [hunk 1]
```diff
@@ -13,45 +13,175 @@ A minimalistic wrapper around React Native's AsyncStorage.
...
+
+----------
+
+ - [Basic Object Handling](#objects)
+ - [Basic Array Handling](#arrays)
+ - [Chaining Methods/Error Handling](#chaining)
+ - [Deleting](#delete)
+
+
+<a name="objects"></a>
```
Comment (jasonmerino) [inline README.md]: Test this out, but I think that Github injects anchors with headings automatically. I think the link for this heading would be `#working-with-objects`. If that would work we could clean this up and not have to worry about adding in the anchors manually.

File: README.md [hunk 2]
```diff
@@ -13,45 +13,175 @@ A minimalistic wrapper around React Native's AsyncStorage.
...
+### Working With Objects
+
+----------
+React-native-simple-store allows you to easily store data by assigning it a unique key. We will show you a few examples of just how easy it is to get started.
+
+
+#### Saving and Retrieval
+
+	// Save an object to a users device. We will assign it a key of 'album' for easy retrieval	   
+    store.save('album', {
```
Comment (jasonmerino) [inline README.md]: This link is one example, but make sure that all indentation on this page is tabs rather than spaces.

File: README.md [hunk 3]
```diff
@@ -13,45 +13,175 @@ A minimalistic wrapper around React Native's AsyncStorage.
...
+	const maleArtist = {
+	   name: 'The Weeknd',
+	   age: 27,
+	   gender: 'male'
+	}
+	
+	store.push('artists', femaleArtist) // Creates a new array, and inserts this object into it.
+	store.push('artists', maleArtist) // Adds this new object to the end of the array.
+	
+	// Our new object will look like this when we retrieve it
```
Comment (jasonmerino) [inline README.md]: Change "object" to "array"

File: README.md [hunk 4]
```diff
@@ -13,45 +13,175 @@ A minimalistic wrapper around React Native's AsyncStorage.
 npm install react-native-simple-store
 ```
 
-## [API Reference](docs/index.md)
```
Comment (DZuz14) [inline README.md]: Whoops, had no idea I left that out, my bad on that one.

**Comment by DZuz14** [comment]
Fixed pretty much everything you asked for but I am going to have to fix the tabs in a little bit, they aren't updating to the 2-tabs that I want. Ill get it fixed.

**Comment by jasonmerino** [comment]
@DZuz14 Thanks for the PR. Docs additions look good so I merged it. :)

**Pull request merged**